cryptol-remote-api/src/CryptolServer.hs

```diff
@@ -77,3 +77,107 @@ validateServerState =
          if fp == Just (lmFingerprint lm)
            then continue
            else return False
+
+--------------------------------------------------------------------------------
+
+-- The schema for translating Cryptol errors/warnings into JSONRPCExceptions
+
+-- Reserved range for Cryptol exceptions: 20,000 - 21,000
+cryptolErrorPrefix :: Integer
```

Comment:
Is this a "prefix", or is it more of a "base"?

Reply:
I'm fine with "base"—changed in 71a7d1d.